server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/step/LoadReportAnalysisMetadataHolderStep.java

```diff
@@ -87,6 +93,25 @@ public void execute() {
     mutableAnalysisMetadataHolder.setOrganization(organization);
   }
 
+  /**
+   * Check that the Quality profiles sent by scanner correctly relate to the project organization.
+   */
+  private void checkQualityProfilesConsistency(ScannerReport.Metadata metadata, Organization organization) {
+    List<String> profileKeys = metadata.getQprofilesPerLanguage().values().stream()
+      .map(QProfile::getKey)
+      .collect(toList(metadata.getQprofilesPerLanguage().size()));
+    try (DbSession dbSession = dbClient.openSession(false)) {
+      List<QualityProfileDto> profiles = dbClient.qualityProfileDao().selectByKeys(dbSession, profileKeys);
+      String badKeys = profiles.stream()
+        .filter(p -> !p.getOrganizationUuid().equals(organization.getUuid()))
+        .map(p -> p.getKey())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612)